e2e/specs/st_file_uploader.spec.ts

```diff
@@ -78,81 +82,109 @@ describe("st.file_uploader", () => {
           }
         );
 
-      cy.get(".uploadError")
+      cy.get(".fileError")
         .first()
-        .should("have.text", " application/json files are not allowedOK");
+        .should("have.text", "application/json files are not allowed.error");
```

Comment:
That `.error` looks a little weird. Can we change the selector so that only `application/json files are not allowed.` is in it?